Supporting Resources/v2.0/Datamodel/Datafiles/Activity I-SD-3-A.yml

```diff
@@ -16,7 +16,7 @@ id:
 
 #The title of this activity
 title:
-  Integrity of the code is verified prior to deployment.
+  Integrity of the binaries is verified prior to deployment.
```

Comment:
Maybe "packages", "artifacts" or "deployables" if code sounds off? I don't think you can consider e.g. JavaScript as binary.

Reply:
Changed into deployables.